bin/MadGraph5_aMCatNLO/cards/production/13TeV/TTToHcToWA_AToMuMu/TTToHcToWA_AToMuMu_MHc160_MA120/TTToHcToWA_AToMuMu_MHc160_MA120_proc_card.dat

```diff
@@ -0,0 +1,22 @@
+import model 2HDM_UFO-type1_5FS
+
+define p = g u d c s b u~ d~ c~ s~ b~
+define j = p
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+definve vl~ = ve~ vm~ vt~
+
+generate p p > t t~, (t > b h+, (h+ > w+ h3, (h3 > mu+ mu-), (w+ > l+ vl))), (t~ > b~ w-, (w- > j j )) @0
+add process p p > t t~, (t > b h+, (h+ > h3 l+ vl, (h3 > mu+ mu-))), (t~ > b~ w-, (w- > j j )) @ 1
+add process p p > t t~,(t > b h+,(h+ > w+ h3,(h3 > mu+ mu-),(w+ > j j))),(t~ > b~ w-,(w- > l- vl~)) @2
+add process p p > t t~, (t > b h+, (h+ > h3 j j, (h3 > mu+ mu-))), (t~ > b~ w-, (w- > l- vl~)) @3
+add process p p > t t~,(t > b h+,(h+ > w+ h3,(h3 > mu+ mu-),(w+ > l+ vl))),(t~ > b~ w-,(w- > l- vl~)) @4
+add process p p > t t~, (t > b h+, (h+ > h3 l+ vl, (h3 > mu+ mu-))), (t~ > b~ w-, (w- > l- vl~)) @5
+add process p p > t t~, (t > b w+, (w+ > j j )), (t~ > b~ h-, (h- > w- h3, (h3 > mu+ mu-), (w- > l- vl~))) @6
+add process p p > t t~, (t > b w+, (w+ > j j )), (t~ > b~ h-, (h- > h3 l- vl~, (h3 > mu+ mu-))) @ 7
+add process p p > t t~, (t > b w+, (w+ > l+ vl)), (t~ > b~ h-, (h- > w- h3, (h3 > mu+ mu-), (w- > j j))) @8
+add process p p > t t~, (t > b w+, (w+ > l+ vl)), (t~ > b~ h-, (h- > h3 j j, (h3  > mu+ mu-))) @9
+add process p p > t t~, (t > b w+, (w+ > l+ vl)), (t~ > b~ h-, (h- > w- h3, (h3 > mu+ mu-), (w- > l- vl~))) @10
+add process p p > t t~, (t > b w+, (w+ > l+ vl)), (t~ > b~ h-, (h- > h3 l- vl~, (h3 > mu+ mu-))) @ 11
```

Comment:
@10 and @11 overlaps
Same for other pairs

Reply:
Thanks, changed.